subgraphs/diva-staking/utils/vaultHandlers.ts

```diff
@@ -0,0 +1,9 @@
+import { createComptroller } from '../entities/Comptroller';
+
+import { ComptrollerLibDataSource } from '../generated/templates';
+import { Address } from '@graphprotocol/graph-ts';
+
+export function handleAccessorSet(nextAccessor: Address, vault: Address): void {
+  createComptroller(nextAccessor, vault);
+  ComptrollerLibDataSource.create(nextAccessor);
+}
```

Comment:
I would remove this file, and just add the two lines in the respective mapping files. Less code, even if duplicated.

Reply:
done